frontend/components/MainPage/sections/AlertSections/UpdateAvailableAlert.tsx

```diff
@@ -1,60 +1,99 @@
 import { useQuery } from '@tanstack/react-query';
-import { Flex } from 'antd';
+import { Flex, Typography } from 'antd';
 import useToken from 'antd/es/theme/useToken';
```

Comment:
I wonder if maybe we can get it directly from theme as we do in frontend/components/ManageStakingPage/StakingContractSection/index.tsx ?